buildSrc/src/main/kotlin/com/github/skydoves/sandwich/Dependencies.kt

```diff
@@ -1,13 +1,13 @@
 package com.github.skydoves.sandwich
 
 object Versions {
-  internal const val ANDROID_GRADLE_PLUGIN = "8.0.0"
-  internal const val ANDROID_GRADLE_SPOTLESS = "6.13.0"
+  internal const val ANDROID_GRADLE_PLUGIN = "8.0.0-rc01"
```

Comment:
The publication issue is fixed by downgrading AGP to `8.0.0-rc01`? Any issue tracker here?

Reply:
Alright, it's something related to Kotlin 1.8.20.

Reply:
Yes, this issue is related to the maven publish gradle plugin and I'm waiting for thier fixes. Rc01 works well.